src/pages/certification/Certification.tsx

```diff
@@ -94,13 +94,11 @@ const Certification = () => {
     }, [lsFormData]);
 
     return (
-        <div className="content-area">
-            <div className="content-area-title-section py-7 flex justify-between">
-                <h2 className="m-0">
-                    {runId
-                        ? "Run: " + ellipsizeString(runId)
-                        : "Testing Tool"}
-                </h2>
+        <Container className="py-4" maxWidth="xl">
```

Comment:
this causes a max-width to content-section which doesn't actually expand the content to fit the screen when >4k. We don't want <Container> elements for that reason. Atleast in Testing and History views. I am reverting this change.